libraries/AP_HAL_ChibiOS/RCOutput.h

```diff
@@ -33,7 +33,7 @@
 #define RCOU_DSHOT_TIMING_DEBUG 0
 
 class ChibiOS::RCOutput : public AP_HAL::RCOutput
-#ifdef HAL_WITH_BIDIR_DSHOT
```

Comment:
This is wrong, its only relevant if bi-dir dshot is enabled

Reply:
Done. But you probably want
'''
#if HAL_WITH_BIDIR_DSHOT
'''
instead of
'''
#ifdef HAL_WITH_BIDIR_DSHOT
'''